dali/pipeline/operator/builtin/external_source.h

```diff
@@ -168,6 +169,7 @@ class ExternalSource : public InputOperator<Backend> {
     layout_ = batch.GetLayout();
   }
 
+  const bool repeats_last_;
```

Comment:
Not necessary, see above.
A more general remark - it's not very useful to have const instance members. There are exceptions (e.g. aggregates that have public fields that should not be modified), but they are rather rare.